### PR TITLE
Update gems and other suprise goodies

### PR DESCRIPTION
**Pull request by dideler**
Shipment Tracker, we haven't forgotten about you :gift_heart:

### CODE REVIEW

**Comment by dideler** [comment]
Was waiting to remove `wip` label until the next version of Rubocop is released which contains fixes for some of the TODOs that have been added, but that's taking too long. Let's get this integrated.

File: Gemfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ gem 'has_secure_token'
 gem 'honeybadger', '~> 2.0'
 gem 'jquery-rails'
 gem 'newrelic_rpm'
-gem 'octokit', '~> 4.0', require: false
+gem 'octokit', '4.1.0', require: false
```
Comment (sgerrand) [inline Gemfile]: :clap:

File: app/models/events/deploy_event.rb [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 module Events
   class DeployEvent < Events::BaseEvent
-    ENVIRONMENTS = %w(uat staging production)
+    ENVIRONMENTS = %w(uat staging production).freeze
```
Comment (sgerrand) [inline app/models/events/deploy_event.rb]: :gift_heart:

**Comment by sgerrand** [comment]
LGTM. :+1:

**Comment by andistuder** [comment]
LGTM - great housekeeping!

**Pull request merged**